package.json

```diff
@@ -40,8 +40,7 @@
     "fsevents"
   ],
   "dependencies": {
-    "@fortawesome/fontawesome": "1.1.5",
-    "@fortawesome/fontawesome-free-solid": "5.0.9",
+    "@fortawesome/fontawesome-free-webfonts": "^1.0.9",
```

Comment:
@jhermsmeier does this mean we depend on internet connection now?

Reply:
Nope, the font itself is in the npm package